pkg/operator/encryption/controllers/condition_controller.go

```diff
@@ -56,6 +58,7 @@ func NewConditionController(
 	return factory.New().WithInformers(
 		kubeInformersForNamespaces.InformersFor("openshift-config-managed").Core().V1().Secrets().Informer(),
 		operatorClient.Informer(),
+		apiServerConfigInformer.Informer(),
```

Comment:
where is that used? (next person will factor this out because apparently it is not referenced anywhere).

Reply:
do you want me to add comment?

Reply:
done